src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Product.xml

```diff
@@ -47,6 +47,8 @@
         <itemOperations>
             <itemOperation name="get">
                 <attribute name="path">products/{id}</attribute>
+                <attribute name="openapi_context">
+                    <attribute name="summary">As a logged administrator use $code to retrieve a product resource. As a Visitor or logged in Customer use $slug to retrieve a product resource.</attribute>                </attribute>
```

Comment:
this closing </attribute> tag has been hidden just like finnish sniper. Can you find it? :D